screens/Budget/BudgetScreen.tsx

```diff
@@ -43,6 +43,7 @@ export default function BudgetScreen({ navigation, route }: RootTabScreenProps<'
     const now = moment();
     const [month, setMonth] = useState(MONTHS_ORDER[now.month()]);
     const [year, setYear] = useState(now.year());
+    const currentMonthIsNotPast = (year > now.year() || (year === now.year() && MONTHS_ORDER.indexOf(month) >= now.month()));

```

Comment:
They should be able to make a budget for the current month so use > not >= (unless now.month is one indexed)

Reply:
Changing the >= to a > would do exactly that. It would exclude the current month from being able to add budgets.